src/connections/ProjectPitch.ts

```diff
@@ -35,24 +32,23 @@ type ProjectPitchConnectionProps = {
 
 const projectPitchConnection: Connection<ProjectPitchConnection, ProjectPitchConnectionProps> = {
   load: ({ projectPitch }, { uuid }) => {
-    if (!projectPitch) projectPitchesGetUUIDIndex({ pathParams: { uuid } });
+    if (projectPitch == null) projectPitchGet({ pathParams: { uuid } });
   },
 
-  isLoaded: ({ projectPitch }) => projectPitch !== undefined,
+  isLoaded: ({ projectPitch }) => projectPitch !== null,
```

Comment:
[As mentioned in my previous review](https://github.com/wri/wri-terramatch-website/pull/1198#discussion_r2085168841), this isn't good enough. Two things:
* Please always use double equals for null comparison. It's always safer to catch both null and undefined since it's so easy for one to sneak in where you expect the other (`!= null` instead of `!== null`). 
* If the request fails, this code will cause the connection continuous try to re-fetch the resource. Please see other connections for an example. The `isLoaded` method pretty much always both checks for existence of the resource, and the fetch failure at a minimum, and the same things need to be checked before calling the API request in the `load()` function.